selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/dashboard/organization/OrganizationPage.java

```diff
@@ -59,12 +56,12 @@
         "//md-content[contains(@class, 'organization-member-list')]//div[contains(@class, 'che-list-header-column')]//span";
     String MEMBERS_LIST_ITEM_XPATH =
         "//md-content[contains(@class, 'organization-member-list')]//span[text() = '%s']";
-    //        String MEMBER_CHECKBOX =
-    // "//div[contains(@class,'che-list-item-checkbox-main')]/md-checkbox[contains(@aria-label,'member')]";
     String MEMBER_CHECKBOX = "//md-checkbox[contains(@aria-label,'member')]";
-    // todo: //span[text()='Delete'] to get Delete button only
     String DELETE_MEMBER_WIDGET_BUTTON = "//che-popup//button";
     String MEMBERS_SEARCH_FIELD = "//div[@class = 'che-list-search']//input";
+    String WORKSPACES_TAB = "//md-tab-item//span[contains(text(), 'Workspaces')]";
```

Comment:
`_XPATH` prefix missed

Reply:
We need to rework all organization page objects - use SeleniumWebDriverHelper, fix locators names. I think it would be better to do it on  a separate issue.